Documentation/writing-principles-and-guidelines/spelling-and-typo3-terminology.rst

```diff
@@ -1,111 +1,1006 @@
+
+    ..  _file-reference:
+    -   *file reference*
+
+        A file reference is a reference to a file stored and used within the
+        TYPO3 system.
+
+    ..  _file-resource:
+    -   *file resource*
+
+        A file resource is a physical file that is stored and managed within the
+        TYPO3 system. A file reference always points to a file resource.
+
+    ..  _file-storage:
+    -   *file storage*
+
+        File storage in TYPO3 manages the organization and storage of files and
+        media resources. Other systems refer to this with a term like
+        "asset storage".
+
+    ..  _fileadmin:
+    -   *fileadmin*
+
+        The :file:`fileadmin` area is a special folder in the TYPO3 backend
+        where files and media resources are managed. This is the default name of
+        file storages since TYPO3 versions, but can be customized.
+
+    ..  _filelist:
+    -   *Filelist*
+
+        The :t3ext:`filelist` is a module in the TYPO3 backend used for
+        displaying and managing files and media resources. It displays contents
+        of all configured file storages. When referencing files from content
+        elements, a popup window will also display the filelist in the backend.
+
+    ..  _flash-message:
+    -   *Flash Message*
+
+        Flash Messages are temporary notifications in the TYPO3 backend that
+        inform users about important events or changes. Also the Extbase
+        framework allows to use an API to display these kinds of messages in the
+        frontend.
+
+    ..  _flexform:
+    -   *FlexForm*
+
+        FlexForms are a selection of functionalities that allow to extend
+        content element settings in the Backend, which can be accessed in the
+        frontend. A flexForm data source (XML format) defines available sheets,
+        sections and fields, which are displayed alongside a record in the
+        backend record editing interface (based on TCA naming).
+        The values entered in a FlexForm data source are also saved as XML data
+        (as some kind of "blob", so it needs serialization and deserialization
+        when being accessed), which allows for any kind of flexible additional
+        data storage in parallel to the relational database tables (like
+        :sql:`tt_content`).
+
+    ..  _fluid:
+    -   *Fluid*
+
+        Fluid is a template engine in TYPO3 used for creating dynamic and
+        customizable frontend layouts. The format looks like HTML and offers
+        specific and extensible tags to be embedded. It also standard variable
+        replacement as well as a large range of algorithmic and logical
+        operations.
+
+    ..  _forge---forger---gerrit:
+    -   *Forge / Forger / Gerrit*
+
+        [Forge](https://forge.typo3.org/) is the central issue platform for
+        letting the Core Team manage the TYPO3 project and its features and
+        bugs. [Forger](https://forger.typo3.com/) and [Gerrit](https://review.typo3.org/)
```

Comment:
Forger is just an Elastic Search representation for faster search and statistics of open/closed tickets, but it does not manage code reviews.

Reply:
"Code review AND management", Forger is a management tool. I think this is okay then?